app/views/rubygems/show.html.erb

```diff
@@ -24,7 +24,7 @@
         <h2 class="gem__ruby-version__heading t-list__heading">
           <%= t '.bundler_header' %>:
           <div class="gem__code-wrap">
-            <input type="text" class="gem__code" id="gemfile_text" value="<%= @latest_version.to_bundler %>" readonly="readonly">
+            <input type="text" class="gem__code" id="gemfile_text" value="<%= @latest_version.to_bundler(locked_version: @on_version_page) %>" readonly="readonly">
```

Comment:
If you usually avoid setting instance vars like that, this an be refactored with a `params` check like so:

```ruby
params[:controller] == "versions" && params[:action] == "show"
```

Just let me know and I'll refactor it.